packages/cli/src/mfa/mfa.service.ts

```diff
@@ -85,12 +86,27 @@ export class MfaService {
 		return await this.authUserRepository.save(user);
 	}
 
-	async disableMfa(userId: string, mfaToken: string) {
-		const isValidToken = await this.validateMfa(userId, mfaToken, undefined);
+	async disableMfaWithMfaCode(userId: string, mfaCode: string) {
+		const isValidToken = await this.validateMfa(userId, mfaCode, '');
```

Comment:
Would be clearer to pass undefined instead of empty string if the value is missing
```suggestion
		const isValidToken = await this.validateMfa(userId, mfaCode, undefined);
```